ocp.tf

```diff
@@ -83,6 +84,7 @@ module "network" {
 }
 
 module "helpernode" {
+  depends_on = [module.bastion]
```

Comment:
Added due to module level synchronization problems when FIPS is enabled.

There is an option to use a small dummy step that uses fips_compliant as the synchronization point only when fips is enabled. We opted for this cleaner approach.